src/controllers/controller.bar.js

```diff
@@ -301,21 +306,23 @@ module.exports = DatasetController.extend({
 					imeta.controller._getValueScaleId() === scale.id &&
 					chart.isDatasetVisible(i)) {
 
-					ivalue = +scale.getRightValue(datasets[i].data[index]);
-					if ((value < 0 && ivalue < 0) || (value >= 0 && ivalue > 0)) {
+                    stackLength = scale._parseValue(datasets[i].data[index]);
```

Comment:
it looks like there are spaces here. should be tabs. you can run `gulp lint` to run a style check